src/views/pages/Analytics/index.vue

```diff
@@ -0,0 +1,243 @@
+            </md-card-header-text>
+          </md-card-header>
+        </md-card>
+    </div>
+    <!-- Graphs -->
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="productTranscations" 
+        title="Milk-Collections-and-Deliveries" 
+        xUnits="Date" 
+        yUnits="Litres"
+      />
+    </md-card>
+    <md-card class='md-elevation-10 graph'>
+      <graph 
+        v-bind:values="moneyTranscations" 
+        title="Loan-Dispersals-and-Payments" 
+        xUnits="Date" 
+        yUnits="UGX"
+      />
+    </md-card>
+  </div>
+</template>
+
+<script>
+import graph from "../../components/AnalyticsGraph";
+import moment from "moment";
+import * as d3 from "d3";
+
+export default {
+  name: "analytics",
+  data() {
+    return {
+      productTranscations: [],
+      moneyTranscations: [],
+      collected: 0,
+      delivered: 0,
+      dispensed: 0,
+    };
+  },
+  components: {
+    graph,
+  },
+
+  created() {
+  // generate data 
+    this.productTranscations.push(this.fillCollections ());
+    this.productTranscations.push(this.fillExports ());
+    this.moneyTranscations.push(this.fillLoans ());
+    this.moneyTranscations.push(this.fillPayments ());
+
+  // calculate the stats
+    this.collected = this.calculateCollected ();
+    this.delivered = this.calculateDelivered ();
+    this.disperesed = this.calculateLoans ();
+  },
+  methods: {
+    fillCollections () {
+      let data = [];
+      for(let i=0; i<40; i++) {
+        let currentDate = new Date("2018-02-20T18:58:51-06:00");
+        currentDate.setDate(currentDate.getDate() + i);
+        let currentValue = d3.randomUniform(10, 200)();
+
+        data.push({
+          type: 'collections',
+          productType: 'milk',
+          datetime: currentDate,
+          amountOfProduct: currentValue,
+          productUnits: 'L',
+          costPerUnit: '54',
+          currency: 'UGX',
+          lastModified: currentValue,
+        });
+      }
+      return data;
+    },
+    
+    fillExports () {
+      let data = [];
+      for(let i=0; i<40; i++) {
+        let currentDate = new Date("2018-02-20T18:58:51-06:00");
+        currentDate.setDate(currentDate.getDate() + i);
+        let currentValue = d3.randomUniform(10, 200)();
+
+        data.push({
+          type: 'deliveries',
+          productType: 'milk',
+          datetime: currentDate,
+          amountOfProduct: currentValue,
+          productUnits: 'L',
+          costPerUnit: '54',
+          currency: 'UGX',
+          lastModified: currentValue,
+        });
+      }
+      return data;
+    },
+    
+    fillLoans () {
+      let data = [];
+      for(let i=0; i<40; i++) {
+        let currentDate = new Date("2018-02-20T18:58:51-06:00");
+        currentDate.setDate(currentDate.getDate() + i);
+        let currentValue = d3.randomUniform(300, 2000)();
+
+        data.push({
+          type: 'loans',
+          datetime: currentDate,
```

Comment:
do we need to/from?

Reply:
the gaph in its current state isn't using that so I didn't include it in the pops passed. in future iterations it might be nice to have in the tooltip when you hover over the node.